SPECS/bpftrace/bpftrace.spec

```diff
@@ -83,6 +82,9 @@ install -p -m 644 tools/*.txt %{buildroot}%{_datadir}/bpftrace/tools/doc
 %{_datadir}/bpftrace/tools
 
 %changelog
+* Tue Apr 08 2025 Sriram Nambakam <snambakam@microsoft.com> - 0.21.0-1
+- Upgrade version to 0.21.0
```

Comment:
consider adding a comment about removing the `bpftrace-0.20-llvm18.patch` patch file

Reply:
Done